src/lib.rs

```diff
@@ -410,6 +412,53 @@ impl Contract {
     pub fn get_community(&self, handle: String) -> Option<Community> {
         self.communities.get(&handle)
     }
+
+    pub fn add_featured_community(&mut self, handle: String) {
```

Comment:
This API would be not easy to adjust sequence of the featured communities. I recommend to set all of the featured communities: `handles: Vec<String>`.

Reply:
Should set all, replace the existing vector of handles or add to it at the end? I am thinking, it should remove the existing one.

Reply:
Also, if this is the case, do we need remove_featured_communities function?

Reply:
Thanks for the clarification, @frol. I have written the code for single function to add/remove/edit, I will pull your latest code and run some tests. Thanks for the Vector -> Vec changes, it was making a few things difficult.

Reply:
Suggested changes done in the latest commit. Sample testnet widget to test - https://test.near.org/devhubopen.testnet/widget/gigs-board.pages.Feed?nearDevGovGigsContractAccountId=contract.devhubopen.testnet